docs/publishers/config.rst

```diff
@@ -167,6 +187,22 @@ loads.
 
      This setting can only be set using :js:func:`window.hypothesisConfig`.
 
+   .. option:: onProfileRequest
+
+     ``function``. A JavaScript function that will be called when the user clicks
+     the user profile (user name) link in the sidebar. No arguments are passed
```

Comment:
I like the username as an argument concept for simplicity sake. But let's keep an eye out for the need to short-circuit the "is this the current user or a different user" question with a secondary argument. It's something they could resolve but we could make it a lot simpler